core/src/main/java/tc/oc/pgm/controlpoint/ControlPointParser.java

```diff
@@ -87,6 +87,8 @@ public static ControlPointDefinition parseControlPoint(
     boolean permanent = XMLUtils.parseBoolean(elControlPoint.getAttribute("permanent"), false);
     float pointsPerSecond =
         XMLUtils.parseNumber(elControlPoint.getAttribute("points"), Float.class, 1f);
+    float pointsBonus =
+        XMLUtils.parseNumber(elControlPoint.getAttribute("bonus-points"), Float.class, 1f);
```

Comment:
The default should probably be 0

Reply:
Thanks for catching that, at the time I did this a few weeks ago I didn't realize this sets the default value.